core/vdbe/mod.rs

```diff
@@ -1754,6 +1759,20 @@ fn exec_substring(
     }
 }
 
+fn exec_typeof(reg: &OwnedValue) -> OwnedValue {
+    match reg {
+        OwnedValue::Null => OwnedValue::Text(Rc::new("null".to_string())),
+        OwnedValue::Integer(_) => OwnedValue::Text(Rc::new("integer".to_string())),
+        OwnedValue::Float(_) => OwnedValue::Text(Rc::new("real".to_string())),
+        OwnedValue::Text(_) => OwnedValue::Text(Rc::new("text".to_string())),
+        OwnedValue::Blob(_) => OwnedValue::Text(Rc::new("blob".to_string())),
+
+        // TODO: Unreachable?
+        OwnedValue::Record(_) => todo!(),
+        OwnedValue::Agg(_) => todo!(),
```

Comment:
Aggregations at least should also be supported:

```
sqlite> select typeof(sum(age)) from users;
integer
sqlite> select typeof(group_concat(age)) from users;
text
```

Dunno if limbo plumbing in general supports it yet but worth a try

Reply:
@travenin limbo should now support e.g. `typeof(group_concat(age))` as well

Reply:
Great to hear. I will craft this forward.